ibeam/src/var.py

```diff
@@ -14,6 +14,29 @@
 CHROME_DRIVER_PATH = os.environ.get('IBEAM_CHROME_DRIVER_PATH')
 """Path to the Chrome Driver executable file."""
 
+SQLITE_DB_PATH = os.environ.get('IBEAM_SQLITE_DB_PATH',
+                               os.path.abspath(
+                                   os.path.join(os.path.abspath(OUTPUTS_DIR), 'ibeam.db')))
+"""Path to the sqlite db file."""
+
+PROXY_SERVER_HTTP = os.environ.get('IBEAM_PROXY_SERVER_HTTP', '0.0.0.0:8080')
+"""Http endpoint to the proxy server."""
+
+PROXY_SERVER_LOG_PATH = os.environ.get('IBEAM_PROXY_SERVER_LOG_PATH',
+                                       os.path.abspath(
+                                           os.path.join(os.path.abspath(OUTPUTS_DIR), 'proxy_server.log')))
```

Comment:
This `os.path.abspath(os.path.join(os.path.abspath(OUTPUTS_DIR),` is used repeatedly in multiple variables- could turn into a function